### ISSUE TITLE
HWND Background Support

### ISSUE THREAD

**Issue by AlexanderQueen** [martinjanas/IdleMinerAutomator#2]
What about making it working in background while discord is minimized? It works only when discord title bar is active.

P.S it get the curently discord window. Also you can't exit the bot if discord windows is minimized, only thru task manager.

**Comment by martinjanas** [comment]
iirc, it should even work when Discord is minimized or not in background.

**Comment by AlexanderQueen** [comment]
Nah, tested and when it minimized it aren't sending the commands.
